modules/quadbin/redshift/sql/QUADBIN_BOUNDARY.sql

```diff
@@ -0,0 +1,18 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION @@RS_PREFIX@@carto.QUADBIN_BOUNDARY
+(quadbin BIGINT)
+RETURNS VARCHAR(MAX)
```

Comment:
Refresh my mind: we don't return a GEOMETRY because Redshift lacks GeoJSON to Geometry conversion?

Reply:
OK, I just saw you comment about it

Reply:
It's because RS doesn't have still `ST_GEOMFROMGEOJSON`. Actually as it's going to be a square we could change the code to return a WKT, until the function arrives in July.

Reply:
It would be nice to have the GEOMETRY already using WKT

Reply:
Updated